### PR TITLE
Add --version option to print version

### PR DESCRIPTION
**Pull request by mikeastock**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -26,6 +28,7 @@ fn main() {
     opts.optopt("R", "", "changes the git remote used (default is origin)", "REMOTE");
     opts.optopt("b", "", "changes the base for merged branches (default is master)", "BRANCH");
     opts.optflag("h", "help", "print this help menu");
+    opts.optflag("", "version", "print the version");
```
Comment (jfrost) [inline src/main.rs]: Don't want to use `-v` as a short option?

Comment (mcasper) [inline src/main.rs]: I think it'd be nice to have the short as well

Comment (mikeastock) [inline src/main.rs]: I hadn't added it due to the potential confusion that `-v` sometimes is verbose but I can add it if you want!

Comment (jfrost) [inline src/main.rs]: I think the common way to go if you add verbose as well is to use `-v` for `verbose` and `-V` for version.

That said, if we don't plan to add verbose, I vote for the lower case `-v`.

Comment (mcasper) [inline src/main.rs]: Good point, I hadn't thought about verbose.

At the moment there's not really any output that I'm silencing, so verbose doesn't seem like a thing that would get added anytime soon

**Pull request merged**